examples/custom-resources/app-protect-waf-bundle/syslog.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
```

Comment:
Is it a requirement to deploy a syslog instance?  Why?

Reply:
removing bundle example